src/effects/native/reverbeffect.cpp

```diff
@@ -26,7 +26,7 @@ EffectManifest ReverbEffect::getManifest() {
     EffectManifestParameter* time = manifest.addParameter();
     time->setId("bandwidth");
     time->setName(QObject::tr("Bandwidth"));
-    time->setName(QObject::tr("Band"));
+    time->setShortName(QObject::tr("BW"));
```

Comment:
@esbrandt what's the latest way to add a hint to translators? Could we address it that way?